earth2studio/models/dx/corrdiff.py

```diff
@@ -329,12 +332,21 @@ def _forward(self, x: torch.Tensor) -> torch.Tensor:
 
         # Create latents
         rnd = StackedRandomGenerator(x.device, sample_seeds)
+
+        # API break: https://github.com/NVIDIA/modulus/pull/540
+        if Version(modulus.__version__) <= Version("0.6.0"):
+            img_resolution_x = self.regression_model.img_resolution
+            img_resolution_y = self.regression_model.img_resolution
+        else:
+            img_resolution_x = self.regression_model.model.img_shape_x
+            img_resolution_y = self.regression_model.model.img_shape_y
+
```

Comment:
I am sort of the opinion that this model should be frozen in time and note expected to be reusable with other domains or checkpoints. In that sense, I think we should simply fix `img_resolution_x` and `img_resolution_y` to be 448 and 448.

Reply:
Good call, I updated it so it just uses the coords to spec this. So now the coords control the latent generation.